Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/overview/adapter/ContactDiaryOverviewAdapter.kt

```diff
@@ -0,0 +1,62 @@
+package de.rki.coronawarnapp.contactdiary.ui.overview.adapter
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import de.rki.coronawarnapp.databinding.IncludeContactDiaryOverviewItemBinding
+import java.util.Locale
+
+class ContactDiaryOverviewAdapter(private val onItemSelectionListener: (ListItem) -> Unit) :
+    RecyclerView.Adapter<ContactDiaryOverviewAdapter.OverviewElementHolder>() {
+    private var _elements = emptyList<ListItem>()
```

Comment:
Minor: Afaik we don't use the _ prefix consistently - drop it?

Reply:
adjusted